Installation instruction updated to fit current progress (#38)
UBC-MDS/autopredictor: README.md

## Installation

Since this package is still in the developing process, it has not been published on PyPI yet. Thus, In order to use this package, please run the instructions provided below.

### Initialization

> 1. Clone this GitHub repository using this command:

Reviewer: Optional:
We might want to add a note about where the command should be run > terminal or or command prompt/Git Bash.

```bash
git clone https://github.com/UBC-MDS/autopredictor.git
```

> 2. Install poetry in your base environment through this [instruction](https://python-poetry.org/docs/#installation).

Reviewer: ```suggestion
2. Install poetry in your base environment by following these [instructions](https://python-poetry.org/docs/#installation).
```

> 3. Run the following commands from the root directory of this project to create an virtual environment for this package and install autopredictor through poetry:

Reviewer 2: ```suggestion
3. Run the following commands from the root directory of this project to create a virtual environment for this package and install autopredictor through poetry:
```

Corrected the "to create a virtual environment"

> 3. Run the following commands from the root directory of this project to create an virtual environment for this package and install autopredictor through poetry:

Reviewer: ```suggestion
3. Run the following commands from the root directory of this project to create a virtual environment for this package and install autopredictor through poetry:
```
